content/news/this-week-in-fluvio-0023.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "This Week in Fluvio #23"
+date: 2022-02-17
+weight: 20
+---
+Welcome to This Week in Fluvio, our weekly newsletter
+for development updates to [Fluvio open source]. Fluvio is a distributed,
+programmable streaming platform written in Rust.
+
+{{< banner >}}
```

Comment:
The text in the We are Hiring banner in that screenshot looks a little bit weird,

Reply:
Font weirdness is a problem on my machine. For what it's worth, if it looks ok to you on [the main site](https://www.fluvio.io/), I haven't made any changes.

Reply:
It also looks weird on my machine https://www.fluvio.io/news/this-week-in-fluvio-0022/
![image](https://user-images.githubusercontent.com/22335041/155235427-13dbeb0e-33b7-4ae8-aaf2-1169da07d9a2.png)